ViewEnvironment/Sources/EnvironmentValues+ViewEnvironment.swift

```diff
@@ -14,16 +14,19 @@
  * limitations under the License.
  */
 
-import SwiftUI
-import WorkflowUI
+#if canImport(SwiftUI)
```

Comment:
The file move an `canImport` are the important bits—happy to revert the formatting changes if we'd prefer to avoid the churn!